data/json/monsters.json

```diff
@@ -2314,13 +2356,82 @@
     "vision_day":40,
     "vision_night":4,
     "death_drops": "default_zombie_death_drops",
+    "upgrades":{
+      "into_group": "GROUP_ZOMBIE_BRUTE",
+      "half_life": 28
+    },
     "hp":120,
     "death_function":["NORMAL"],
     "special_attacks":[["SMASH", 30], ["GRAB", 7]],
     "description":"Its entire body bulges with distended muscles and swollen, festering wounds.",
     "flags":["SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "REVIVES", "BONES", "PUSH_MON"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_zombie_brute_ninja",
+    "name": "zombie nightstalker",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"dkgray_red",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":24,
+    "aggression":100,
+    "morale":100,
+    "speed":110,
+    "melee_skill":8,
+    "melee_dice":5,
+    "melee_dice_sides":4,
+    "melee_cut":4,
+    "dodge":4,
+    "armor_bash":10,
+    "armor_cut":8,
+    "vision_day":15,
+    "vision_night":40,
+    "death_drops": "default_zombie_death_drops",
+    "hp":220,
+    "death_function":["NORMAL"],
+    "special_attacks":[["SMASH", 30], ["STRETCH_ATTACK", 3], ["LONGSWIPE", 15]],
+    "description":"Somehow this brute hides in the dark like some kind of boogeyman.  Very agile for such a large zombie.",
+    "flags":["SEES", "HEARS", "GOODHEARING", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "REVIVES", "BONES", "PUSH_MON", "NIGHT_INVISIBILITY"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_zombie_brute_grappler",
+    "name": "zombie wrestling brute",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"red_white",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":24,
+    "aggression":100,
+    "morale":100,
+    "speed":95,
+    "melee_skill":7,
+    "melee_dice":3,
+    "melee_dice_sides":8,
+    "melee_cut":1,
+    "dodge":3,
+    "armor_bash":10,
+    "armor_cut":14,
+    "armor_stab":8,
+    "vision_day":40,
+    "vision_night":3,
+    "death_drops": "default_zombie_death_drops",
+    "hp":280,
+    "death_function":["NORMAL"],
+    "special_attacks":[["SMASH", 30], ["BIO_OP_TAKEDOWN", 20], ["RANGED_PULL", 20], ["GRAB_DRAG", 10],
+        {
+        "type" : "bite",
+        "cooldown" : 15,
+        "no_infection_chance" : 5,
+        "min_mul" : 0.5, "//" : "stronger jaws"
+        }],
```

Comment:
At this point the default bite attack is pretty much a waste of turn. Regular brute doesn't bite.